engine/src/main/java/org/destinationsol/menu/MainMenuScreen.java

```diff
@@ -26,6 +26,7 @@
 import org.destinationsol.assets.audio.OggMusicManager;
 import org.destinationsol.common.SolColor;
 import org.destinationsol.game.DebugOptions;
+import org.destinationsol.menu.background.MenuBackgroundManager;
```

Comment:
What's this doing here?

Reply:
Residue that I forgot to clean after shifting stuff to SolApplication, removed it now.